libraries/AP_Scripting/examples/ahrs-source-gps-wheelencoders.lua

```diff
@@ -67,9 +63,7 @@ function update()
   local gps_over_threshold = (gps_speed_accuracy == nil) or (gps:speed_accuracy(gps:primary_sensor()) > gps_speedaccuracy_thresh)
 
   -- get GPS innovations from ahrs
-  local gps_innov = Vector3f()
-  local gps_var = Vector3f()
-  gps_innov, gps_var = ahrs:get_vel_innovations_and_variances_for_source(3)
+  local gps_innov = ahrs:get_vel_innovations_and_variances_for_source(3)
```

Comment:
no need for a "_" to catch the second Vector3f that is returned?

Reply:
We can safely discard it, either method is OK.